### PR TITLE
Pages encoding followup

### PR DESCRIPTION
**Pull request by dannon**
Removes another unnecessary unicodify in pages as a followup to #2197.

### CODE REVIEW

**Comment by dannon** [comment]
As far as I can tell, the contents here are always already unicode in and unicode out as well, so there's no need for the extra unicodify.  Does anyone know of a situation where this isn't the case?

File: lib/galaxy/webapps/galaxy/controllers/page.py [hunk 1]
```diff
@@ -625,11 +624,9 @@ def display_by_username_and_slug( self, trans, username, slug ):
             else:
                 user_item_rating = 0
         ave_item_rating, num_ratings = self.get_ave_item_rating_data( trans.sa_session, page )
-
         # Output is string, so convert to unicode for display.
```
Comment (nsoranzo) [inline lib/galaxy/webapps/galaxy/controllers/page.py]: The comment here should be removed.

Comment (dannon) [inline lib/galaxy/webapps/galaxy/controllers/page.py]: Yep.  Updating, thanks.

**Comment by nsoranzo** [comment]
`sanitize_html()` function is calling `p.feed(unicodify(htmlSource, encoding))`, while `display_by_username_and_slug()` method calls `processor.feed( page.latest_revision.content )` without unicodifying, so I am not completely sure this is safe.

**Comment by dannon** [comment]
@nsoranzo Yep, it's a different code path than the other PR for sure.  In my testing, though, the contents were still unicode on both ends -- are you seeing differently?

**Comment by nsoranzo** [comment]
It seems that `page.latest_revision.content` is always unicode and `processor.output()` is of type `str` only when it is the empty string.

**Pull request closed without merging**